streaming/vision/convert/README.md

```diff
@@ -0,0 +1,119 @@
+    ```bash
+    ├── annotations
+    │   ├── training
+    │   └── validation
+    └── images
+        ├── training
+        └── validation
+    ```
+
+3. Run the [ade20k.py](ade20k.py) script as shown below with the minimalist arguments. The script converts the `train` and `val` dataset split into their own directory. For advanced users, you can look at the supported arguments for [ade20k.py](ade20k.py) and change according to your own needs.
+      <!--pytest.mark.skip-->
+      ```
+      python ade20k.py --in_root <Above directory> --out_root <local or remote directory path to save output MDS shard files>
+      ```
+
+## 2. [CIFAR10](https://www.cs.toronto.edu/~kriz/cifar.html)
+
+**Instructions:**
+
+1. Run the [cifar10.py](cifar10.py) script as shown below with the minimalist arguments which also downloads the raw CIFAR10 dataset if it doesn't exist. For advanced users, you can look at the supported arguments for [cifar10.py](cifar10.py) and change according to your own needs.
+    <!--pytest.mark.skip-->
+    ```
+    python cifar10.py --in_root <Above directory> --out_root <local or remote directory path to save output MDS shard files>
+    ```
+
+## 3. [MS-COCO](https://cocodataset.org/#home)
+
+**Instructions:**
+
+1. Download the COCO 2017 dataset from [here](https://cocodataset.org/#download). Please download both the COCO images and annotations and unzip the files as shown below.
+    <!--pytest.mark.skip-->
+    ```bash
+    mkdir coco
+    wget -c http://images.cocodataset.org/annotations/annotations_trainval2017.zip
+    wget -c http://images.cocodataset.org/zips/train2017.zip
+    wget -c http://images.cocodataset.org/zips/val2017.zip
+
+    unzip annotations_trainval2017.zip
+    unzip train2017.zip
+    unzip val2017.zip
+
+    rm annotations_trainval2017.zip
+    rm train2017.zip
+    rm val2017.zip
+    ```
+
+    That will result in this directory structure:
+    <!--pytest.mark.skip-->
+    ```bash
+    ├── annotations
+    │   ├── instances_train2017.json
+    │   └── instances_val2017.json
+    ├── train2017
+    │   ├── 000000391895.jpg
+    |   |── ...
+    └── val2017
+    │   ├── 000000000139.jpg
+    |   |── ...
+    ```
+
+2. Run the [coco.py](coco.py) script as shown below with the minimalist arguments. The script converts the `train` and `val` dataset split into their own directory. For advanced users, you can look at the supported arguments for [coco.py](coco.py) and change according to your own needs.
+    <!--pytest.mark.skip-->
+    ```
+    python coco.py --in_root <Above directory> --out_root <local or remote directory path to save output MDS shard files>
+    ```
+
+## 4. [ImageNet](https://www.image-net.org/)
+
+**Instructions:**
+
+1. Download the ImageNet dataset from [here](https://image-net.org/download.php). More specifically, 2 files would be needed, `ILSVRC2012_img_train.tar` for training and `ILSVRC2012_img_val.tar` for validation. Then, untar both the files as shown below.
```

Comment:
```suggestion
1. Download the ImageNet dataset from [here](https://image-net.org/download.php). Two files are needed, `ILSVRC2012_img_train.tar` for training and `ILSVRC2012_img_val.tar` for validation. Next untar both the files as shown below.
```

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/183/commits/4d242aaf5f26f73278a555a3021da6e328d0a28b.